frontend/server/src/DAO/Contests.php

```diff
@@ -634,6 +634,113 @@ final public static function getContestsParticipating(
         ];
     }
 
+    /**
+     * Returns the next contest a user registered for.
+     *
+     * @return ContestListItem
+     */
+    final public static function getNextRegisteredContestForUser(
+        int $identityId
```

Comment:
En caso de que se elimine la función que habías creado en el controller, es mejor acá mandar como parámetro todo el objeto de la identidad:
```suggestion
        ?\OmegaUp\DAO\VO\Identities $identity
```
Aún seguiría aceptando null